Defs/Stats/Stats_Basics_Inventory.xml

```diff
@@ -11,5 +11,15 @@
     <toStringStyle>FloatTwo</toStringStyle>
     <displayPriorityInCategory>10</displayPriorityInCategory>
   </StatDef>
+  
+  <StatDef>
+    <defName>AmmoSet</defName>
+    <workerClass>CombatExtended.StatWorker_AmmoSet</workerClass>
+    <label>ammo set</label>
```

Comment:
Calling it ammo set is inconsistent with the caliber stat and very gamey. Should be caliber. Also copy paste description should be solved in a more robust manner, e.g. def inheritance.